src/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -68,11 +68,24 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
         <SeoDataForArtwork artwork={artwork} />
 
         {this.renderImageMetaTags()}
+        {this.maybeRenderNoIndex()}
 
         <ArtworkZendeskFragmentContainer artwork={artwork} />
       </>
     )
   }
+
+  maybeRenderNoIndex() {
+    const { artwork } = this.props
+
+    if (artwork?.visibilityLevel == "UNLISTED") {
+      return (
+        <>
+          <Meta name="robots" content="noindex, follow" />
+        </>
+      )
+    }
+  }
```

Comment:
Following the pattern we were using for Artist meta
https://github.com/artsy/force/blob/main/src/Apps/Artist/Components/ArtistMeta.tsx#L187-L197